src/components/selects-request/archive-list/table/colums.tsx

```diff
@@ -0,0 +1,80 @@
+import moment from 'moment';
+
+import { P } from '@src/component/atoms';
+import { SelectsRequestStatusToKor } from '@src/types';
+import { getUploaderTypeIconComponent } from '@src/lib/User';
```

Comment:
```suggestion
import { P } from '@src/component/atoms';

import { getUploaderTypeIconComponent } from '@src/lib/User';
import { SelectsRequestStatusToKor } from '@src/types';
```

import문 줄바꿈/순서 변경해야합니다~

Reply:
9215a4e